### ISSUE TITLE
Quick Start Guide

### ISSUE THREAD

**Issue by wbuchanan** [wbuchanan/sdpConvening2017#37]
Do we want to create the quick start type guide that we were asked about in a Wiki page or as a separate document that we would include in the repository?  I think what @nathant23 included in his part of the workshop provided fairly good coverage, so I'm not sure exactly what more people may have been looking for.

**Comment by nathant23** [comment]
From what I remember, I believe they were looking for what is the minimal thing they could do to implement git and GitHub in their organization and still get a lot of value.  The goal being how can they get started without being overwhelmed or getting stuck because they think they need from IT that they can't readily get.  
That is when @jsonbecker started suggesting that they don't need to worry about branches and could get a lot of value from just using the master branch and having detailed commits. Is that inline with what you remember @jsonbecker?

**Comment by jsonbecker** [comment]
Yup, I think it was just a doc from me on "minimal git/github" and what you can get out of it. I can draft tomorrow.
